src/main/flight/mixer.c

```diff
@@ -363,19 +363,21 @@ static void applyRpmLimiter(mixerRuntime_t *mixer)
     // Throttle limit learning
     if (error > 0.0f && rcCommand[THROTTLE] < rxConfig()->maxcheck) {
         mixer->rpmLimiterThrottleScale *= 1.0f - 4.8f * pidGetDT();
-    } else if (pidOutput < -400 * pidGetDT() && rcCommand[THROTTLE] >= rxConfig()->maxcheck && !areMotorsSaturated()) { // Throttle accel corresponds with motor accel
+    } else if (pidOutput < -400.0f * pidGetDT() && rcCommand[THROTTLE] >= rxConfig()->maxcheck - 5 && !areMotorsSaturated()) { // Throttle accel corresponds with motor accel
```

Comment:
Please explain the `-5` in comment for reference.

Reply:
"magic" numbers should really be defined constants

Reply:
Good catch !

Reply:
The commit name that introduced the `-5` gives a hint:
> [fix for maxcheck condition not satisfied if set to 2000](https://github.com/betaflight/betaflight/pull/13039/commits/97b261c66979b98c7834f2dfc0d2a81b9d316eac)

`rcCmmand[THROTTLE]` is a float, whereas `maxcheck` is an integer. I wonder if this can be fixed more gracefully. Maybe like this?
```
lrintf(rcCommand[THROTTLE]) >= rxConfig()->maxcheck
```

Reply:
-1 would be enough for rounding, but maybe small deadband correction?

Reply:
-1 would be enough for rounding, but maybe small deadband correction?

@Tdogb any specific reason using 5 ?

Reply:
If it's a deadband correction this is an issue of the gimbal in Tdogb's radio and should be fixed in CLI/configurator (in the radio settings).
![grafik](https://github.com/betaflight/betaflight/assets/19867640/ff41469e-165b-4f58-917b-6b59ffd271dd)

If you actually want to check for the throttle cap instead of when throttle is considered to be "HIGH", then you need another variable to check for (max_throttle).

Reply:
I subtract 5 to add a deadband because MANY pilots set their max_check parameter to 2000 and then the rpm limiter never moves the throttle cap up. It's actually one of the most complained about "issues" in the field.

You bring up a good point however...Maybe this should be changed to maxthrottle. I really just want to check if the user is at (or very close to) full throttle.